webpack.config.js

```diff
@@ -0,0 +1,33 @@
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const path = require('path');
+
+module.exports = {
+  mode: 'development',
+  entry: {
+    index: './src/index.js',
+    Leaderboard: './src/Leaderboard.js',
+    Score: './src/Score.js',
```

Comment:
[OPTIONAL] I think we don't need to call the `Leaderboard` and the `Score` files in the webpack config. It will automatically parse every file that it's somehow linked (imported) to our entry file `index.js`. But this is a bit beyond my knowledge. I would like to know your reasons as to why are they're here.

Reply:
Hello! This was a misunderstanding on my part regarding how to manage multiple entry points. This is technically valid, but it is more for apps with a separate vendor entry and for multiple page applications(see https://webpack.js.org/concepts/entry-points/#scenarios)
This fix is coming on my next attempt.